forge-gui/res/cardsfolder/upcoming/afterlife_insurance.txt

```diff
@@ -0,0 +1,7 @@
+Name:Afterlife Insurance
+ManaCost:1 WB
+Types:Instant
+A:SP$ PumpAll | Cost$ 1 WB | ValidCards$ Creature.YouCtrl | KW$ Afterlife:1 | SubAbility$DBDraw  | SpellDescription$ Creatures you control gain afterlife 1 until end of turn. Draw a card.
```

Comment:
`| SubAbility$ DBDraw | SpellDescription$ Creatures you control gain afterlife 1 until end of turn.`

more readable with the spaces, removed "Draw a Card" Spell Description because the other line already has it